contracts/ArrayLib.sol

```diff
@@ -129,6 +129,34 @@ library UInt256ArrayLib {
       }
     }
   }
+
+  /**
+   * @notice Shorten an array to specified length.
+   * @dev Shortens the specified array to the specified length by directly overwriting
+   * the length of the original array in storage.
+   *
+   * ##### Example usage:
+   *
+   * ```solidity
+   * new uint256[](100).fill(1).cutToLength(50); // resizes the original array to length 50
+   * ```
+   * -
+   * @param values The array to shorten.
+   * @param length The desired length of the array.
+   * @return values The shortened array.
+   */
+  function cutToLength(
```

Comment:
```suggestion
  function shrink(
```

Reply:
also, do we still use .slice elsewhere? If not, we should delete that from ArrayLib.

Reply:
per our discussion -- we're going to keep `slice` because it's used in some tests and it's already audited and may be useful in the future